code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1980,15 +1980,30 @@ mob/living/carbon/human/isincrit()
 	else
 		return ..()
 
+/mob/living
+	var/hangman_score = 0 // For round end leaderboards
+
 /mob/living/carbon/human/Hear(var/datum/speech/speech, var/rendered_speech="")
 	..()
 	if(stat)
 		return //Don't bother if we're dead or unconscious
 	if(ear_deaf || speech.frequency || speech.speaker == src)
 		return //First, eliminate radio chatter, speech from us, or wearing earmuffs/deafened
+	var/mob/living/H = speech.speaker
+	if(muted_letters && muted_letters.len && length(speech.message) == 1)
```

Comment:
Guessing with any punctuation (e.g.: `A!`) will count as a non-guess

Reply:
Maybe I'll try strip that from speech messages or something.

Reply:
Done